inst/apphelp/descriptionCnaTab.md

```diff
@@ -0,0 +1,26 @@
+**Load a valid copy number data file.** This file is used in cBioPortal for the copy number data panel (see image on the right.)
+
+---
+
+#### A minimal copy number data file with only the columns required for cBioPortal:
+
+The discrete copy number data file contain values that would be derived from copy-number analysis algorithms like GISTIC 2.0 or RAE. For each gene (row) in the data file, the following columns are required in the order specified:
+
+- **Hugo_Symbol (Required)**: A HUGO gene symbol.
+
+Next to Hugo_Symbol, it is recommended to have the Entrez gene ID:
+
+- **Entrez_Gene_Id (Optional, but recommended)**: An Entrez Gene identifier.
+
+For each sample in the dataset an additional column is required using the sample id as the column header:
```

Comment:
We have to mention that the sample ID should be already present in the sample tab.